modules/jolt_physics/spaces/jolt_layers.cpp

```diff
@@ -191,6 +191,8 @@ JoltLayers::JoltLayers() {
 	_allocate_object_layer(0);
 }
 
+JoltLayers::~JoltLayers() {}
```

Comment:
You should be able to get away with this, I think:

```suggestion
JoltLayers::~JoltLayers() = default;
```